docs/shrc.md

```diff
@@ -66,43 +140,138 @@ The [-Xshareclasses](xshareclasses.md) option is highly configurable, allowing y
 
     In addition, if you have VMs from different Java installations, of the same Java release and installed by the same user, each VM checks whether the existing default shared cache in the cache directory is from the same Java installation as the VM. If not, the VM deletes that shared cache, then creates a new one. Specifying a different cache directory for each Java installation avoids this situation.
 
-
-
 - Ensure that the cache directory permissions are set appropriately ([`-Xshareclasses:cacheDirPerm`](xshareclasses.md#cachedirperm)).
 
-    It is good practice to explicitly set permissions for the cache directory when the defaults are not appropriate.
+    It is good practice to explicitly set permissions for the cache directory when the defaults are not appropriate. Access is controlled by operating system permissions and Java security permissions; read/write access is the default and gives all users equal rights to update the cache. On Unix systems, you can use the [`-Xshareclasses:groupAccess`](xshareclasses.md#groupaccess) suboption to allow read/write permissions for groups as well as users.
+
+    On z/OS, a cache can be accessed only by a VM that is running in the same storage key as the VM that created the cache. If the keys do not match, permission to access the cache is denied.
 
 - Set the [`-Xshareclasses:nonfatal`](xshareclasses.md#nonfatal) option.
 
-    This option means that your application can start even if there is a problem opening or creating the shared cache, in which case, the VM might be able to start *without* class data sharing.
+    Setting this option allows your application to start even if there is a problem opening or creating the shared cache. In this situation, the VM might be able to start *without* class data sharing.
+
+When the cache is active, you can adjust the soft maximum size by using the [`-Xshareclasses:adjustsoftmx`](xshareclasses.md#adjustsoftmx) cache utility.
```

Comment:
This should be another bullet point. It seems sparse, as soft max hasn't been introduced.

Reply:
I've moved it into a bullet about setting a soft maximum size, and linked to the `-Xscmx` topic where we have more information.